test.py

```diff
@@ -0,0 +1,94 @@
+#fasta
+fasta = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
```

Comment:
ここ、中身をいじっちゃダメな定数として扱っていいかも
定数は
```suggestion
FASTA = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
```
大文字で表すと良い